pages/starting.tsx

```diff
@@ -0,0 +1,16 @@
+import { InferGetStaticPropsType, GetStaticProps } from 'next';
+
+import { Page, PageSidebar } from '@patternfly/react-core';
+
+import NavExpandableList from '@components/starting/SideNavigation';
+
+/**
+ * Renders the {@link HomePage}
+ */
+const HomePage: React.FC<InferGetStaticPropsType<GetStaticProps>> = () => (
+  <div>
+    <Page sidebar={<PageSidebar nav={<NavExpandableList />} />}></Page>
+  </div>
+);
+
+export default HomePage;
```

Comment:
should this be called something other than the HomePage? You already have `HomePage` in `pages/index.tsx`